### ISSUE TITLE
Drag and Drop after last element.

### ISSUE THREAD

**Issue by CodingJenky** [glekli/jQuery-Sortable-Photos#1]
Hello. In your demo for sorting photos I am unable to drag a photo to the end of the photo list. It will either go before or back to where it was.

In my scenario it was one image per row so a simple fix was updating the _onDrop function as follows:

 _onDrop: function(droppable, ui) {
            // Move the dragged item to the new position in the DOM
            // and rearrange the items.
            // Need to use .detach() to keep the event listeners added
            // by jQuery UI.
            var draggable = $(ui.draggable);  
            var movedItem = draggable.detach();

```
      //put if else check for movedItem.insertBefore() where it only puts the draggable object after
     //if its row id is less than the droppable object's row id
        if(draggable.attr("data-row-id") < $(droppable).attr("data-row-id")){
            movedItem.insertAfter(droppable);
        }
        else{
            movedItem.insertBefore(droppable);
        }
        this.arrange();

        // Trigger event so that third parties have a chance to process the result.
        this._trigger('afterDrop', null, this.element);
    }
```

Hope this helps. Thanks for the library.

![image](https://cloud.githubusercontent.com/assets/1840980/12503299/35c79796-c127-11e5-9d85-96c2855966dc.png)

**Comment by glekli** [comment]
Thanks for posting this. Great idea to address the issue. I will consider including this in the next release.

**Comment by Devdomondo** [comment]
This is a great plugin. There is one thing I cannot figure out. How can I define how many photos per row?
Hope you can help me.
Thanks,
Roberto
